src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -95,6 +95,13 @@ def validate(self, attrs):
         # project_rule(_details) endpoints by setting it on attrs
         actions = attrs.get("actions", tuple())
         for action in actions:
+            # XXX(colleen): For rules that have the Jira integration as an action
+            # we need to ensure the user has selected a Jira project and issue type
+            if action["id"] == "sentry.integrations.jira.notify_action.JiraCreateTicketAction":
+                if not action.get("issuetype") and not action.get("project"):
```

Comment:
@ceorourke Pretty sure you want an `OR` condition here since we need both fields set